pkg/loop/internal/relayer/relayer.go

```diff
@@ -336,21 +347,24 @@ func (r *relayerServer) NewConfigProvider(ctx context.Context, request *pb.NewCo
 }
 
 func (r *relayerServer) NewPluginProvider(ctx context.Context, request *pb.NewPluginProviderRequest) (*pb.NewPluginProviderReply, error) {
-	exJobID, err := uuid.FromBytes(request.RelayArgs.ExternalJobID)
+	reqRelayArgs := request.RelayArgs
+	reqPluginArgs := request.PluginArgs
```

Comment:
nit: as these are local variables, consider rargs, pargs since that's more idiomatic :)

Reply:
I'm a recovering java coder and struggle with brevity of naming ;-) will shorten